spokestack/vad/webrtc.py

```diff
@@ -1,13 +1,12 @@
 """
-This module contains the webrtc component for
-voice activity detection (vad)
+This module contains the webrtc component for voice activity detection (vad)
```

Comment:
it is a bit weird to have a separate wrapper module/class for the vad stuff and not for the other webrtc components
since the client won't see it, I suppose this is fine, but if you can find a way to make them all consistent, that would be ideal

Reply:
yeah. that makes sense. I definitely like the way the vad is setup vs the other extensions.